kolla/defaults.yml

```diff
@@ -369,6 +369,11 @@ enable_usage_reporting: no
 # image tools
 enable_image_deployer: no
 enable_image_cleaner: no
+image_container: chameleon-supported-images
+image_type: qcow2
+image_prefix: testing_
+image_scope: prod
+image_object_store_url: https://chi.tacc.chameleoncloud.org:7480/swift/v1/AUTH_key
```

Comment:
shouldn't this be:
```
image_object_store_url: https://chi.tacc.chameleoncloud.org:7480/swift/v1/AUTH_f6c7696906c04b3c89fc3bda9a1b8be0
```

Reply:
Yeah, I missed updating it later after testing that locally.